apps/08jquery-towers-of-hanoi/script.js

```diff
@@ -1,5 +1,32 @@
 'use strict';
 
+var $block = null;
+//This below gets the document ready for jQuery
 $(document).ready(function() {
-  // Put app logic here
+  $('[data-stack]').on('click', function() {
+    //Code below initiates the game to be played with they click on it
+    if ($block === null) {
+      $block = $(this).children().last().detach();
+    } else {
+      if (
+        //this takes a block and appends it to the next 
+        $($block).data('block') < $(this).children().last().data('block') ||
+        $(this).children().length === 0) {
+        $(this).append($block);
```

Comment:
This is probably the stuff I'd be commenting on more so than e.g. the checkForWin comments. Think about things that are confusing for a reader (or grader 😄 )